src/components/sidebarInternals.svelte

```diff
@@ -72,312 +72,396 @@
-											{/if}
-										</p>
-										{#if option.data.vehicle_id && !(option.data.chateau_id == 'san-diego-mts' && option.data.route_type == 0)}
-											<p class="text-sm lg:text-base">{$_('vehicle')} {option.data.vehicle_id}</p>
+									<p class="text-sm lg:text-base mt-0">
+										{#if option.data.headsign && option.data.headsign != option.data.route_long_name && option.data.headsign != option.data.route_short_name}
+											{'→'}
+											<span class=""
+												>{fixHeadsignText(
+													option.data.headsign,
+													option.data.route_short_name || option.data.route_long_name
+												)}</span
+											>
 										{/if}
-									</div>
-								{/each}
-							</div>
-						{/if}
+										{#if fixHeadsignIcon(option.data.headsign)}
+											<span class="material-symbols-outlined text-sm align-middle"
+												>{fixHeadsignIcon(option.data.headsign)}</span
+											>
+										{/if}
+									</p>
+									{#if option.data.vehicle_id && !(option.data.chateau_id == 'san-diego-mts' && option.data.route_type == 0)}
+										<p class="text-sm lg:text-base">{$_('vehicle')} {option.data.vehicle_id}</p>
+									{/if}
+								</div>
+							{/each}
+						</div>
+					{/if}
 
-						{#if latest_item_on_stack.data.arrayofoptions.filter((x) => x.data instanceof RouteMapSelector).length > 0}
-							<h3 class="text-base sm:text-lg">{$_('routes')}</h3>
-							<p>Selecting routes doesn't do anything yet!</p>
-							<div class="flex flex-col gap-y-1 md:gap-y-2">
-								{#each latest_item_on_stack.data.arrayofoptions.filter((x) => x.data instanceof RouteMapSelector) as option}
-									<div
-										class="px-1 py-0.5 md:px-2 md:py-2 bg-gray-50 dark:bg-slate-800 shadow-md shadow-gray-500 dark:shadow-slate-700 text-sm md:text-base"
-									>
+					{#if latest_item_on_stack.data.arrayofoptions.filter((x) => x.data instanceof RouteMapSelector).length > 0}
+						<h3 class="text-base sm:text-lg pt-2">{$_('routes')}</h3>
+						<p>Selecting routes doesn't do anything yet!</p>
+						<div class="flex flex-col gap-y-1 md:gap-y-2 pt-2">
+							{#each latest_item_on_stack.data.arrayofoptions.filter((x) => x.data instanceof RouteMapSelector) as option}
+								<div
+									class="flex items-center justify-between px-1 py-0.5 md:px-2 md:py-2 bg-gray-50 dark:bg-slate-800 text-sm md:text-base rounded-md"
+								>
+									<span>
 										<p>{option.data.chateau_id}</p>
 										{#if option.data.name}
 											<span
 												style={`color: ${darkMode ? lightenColour(option.data.colour) : option.data.colour}`}
 												>{option.data.name}</span
 											>
 										{/if}
-									</div>
-								{/each}
-							</div>
-						{/if}
-						<br />
-						<br />
-						<br />
-					</div>
+									</span>
+									{#if is_schedule_pdf(option.data.chateau_id, option.data.route_id)}
+										<a
+											style:cursor="pointer"
+											on:click={async () => {
```

Comment:
This is bad design, it should fill the href. if it's complicated, make a seperate component to contain the logic there.